### PR TITLE
add eslint

### PR DESCRIPTION
**Pull request by joelhooks**
am I the only one that gives a shit about the rules???

### CODE REVIEW

File: .eslint.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+module.exports = {
```
Comment (joelhooks) [inline .eslint.js:1]: used this as a reference: https://paulintrognon.fr/blog/post/typescript-prettier-eslint-next-js

**Comment by jh3y** [comment]
The perfect GIF! 😅

**Comment by jh3y** [comment]
Looks good to me 👍  But are we missing `prettierrc`?

**Comment by joelhooks** [comment]
@jh3y it's stuffed in package.json I believe

**Comment by jh3y** [comment]
Cool! 😎 

Purely asked as it's referenced in the linting configuration 👍

**Pull request merged**